apps/prairielearn/src/pages/administratorJobSequence/administratorJobSequence.html.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function administratorJobSequence({
+  job_sequence,
+  resLocals,
+}: {
+  job_sequence: any;
+  resLocals: Record<string, any>;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head') %>", {
+          ...resLocals,
+          pageTitle: `${job_sequence.description} #${job_sequence.number}`,
+        })}
+        <script src="<%= node_modules_asset_path('socket.io-client/dist/socket.io.min.js') %>"></script>
```

Comment:
Rather than using EJS syntax here, this should directly use the `nodeModulesAssetPath` function from `lib/assets`. Here's an example:

https://github.com/PrairieLearn/PrairieLearn/blob/bec18b120208ad4c482ce7a7aedf2bbb4f2ea294/apps/prairielearn/src/components/QuestionsTable.html.ts#L11

Reply:
Resolved in commit 6f57e22